client/src/languages/rPython.ts

```diff
@@ -0,0 +1,327 @@
+import {h} from 'maquette';
+import * as Langs from '../definitions/languages'; 
+import * as Graph from '../definitions/graph'; 
+import * as Values from '../definitions/values'; 
+import {printPreview} from '../editors/preview'; 
+import {createEditor} from '../editors/editor';
+import {Md5} from 'ts-md5';
+import axios from 'axios';
+
+// declare var PYTHONSERVICE_URI: string;
+// declare var RSERVICE_URI: string;
+declare var DATASTORE_URI: string;
+
+// ------------------------------------------------------------------------------------------------
+// rPython plugin
+// ------------------------------------------------------------------------------------------------
+
+/// A class that represents a Python block. All blocks need to have 
+/// `language` and Python also keeps the Python source we edit and render
+
+type languageType = "python" | "r"
```

Comment:
We should not restrict this to R and Python - anybody who can provide a URL for another service should be able to use this class (without modifying it) to create a new language plugin for their language.

Reply:
agree, I will remove this